pkg/eventing/v1/client.go

```diff
@@ -148,7 +150,7 @@ func (c *knEventingClient) UpdateTriggerWithRetry(ctx context.Context, name stri
 }
 
 func updateTriggerWithRetry(ctx context.Context, c KnEventingClient, name string, updateFunc TriggerUpdateFunc, nrRetries int) error {
-	b := retry.DefaultRetry
+	b := config.DefaultRetry
```

Comment:
Looks good to me, except that I probably would change the signature/API instead of taking a retry number to take a full backoff struct for more flexibility.

But happy to merge now, too, we could adapt later as well.

Reply:
Since the UpdateServiceWithRetry function is being used in the func plugin as well, I think we can change the signature later?